### ISSUE TITLE
first blog post

### ISSUE THREAD

**Issue by stevep2007** [GDEED/RV_Root#194]
Here is the first blog post. 

https://docs.google.com/document/d/1lm38PjK7CEQeJafp7x55hmy-_n0aGtnv06gKbu6uLDw/edit

And here is an image to run with it on top of the post. 

Also if you need a header for the blog, please use the same image as the one on the apply page.

**Comment by cwrage77** [comment]
Hey steve, there was supposed to be a mark down for this.

Can't give you the image support at this time, unless you want to put it on the top banner, please review at 
https://realityvirtuallyhack.com/blog/first-post

please review and provide feedback

**Comment by mcclaskiem** [comment]
We might be able to drop a simple inline image tag and I can add some styling to make it responsive

**Comment by cwrage77** [comment]
Yeah that sounds good, I believe we need some custom metadata for the fb posts etc, looks ok on g+ / tw otherwise, @stevep2007 has details..

**Comment by cwrage77** [comment]
> We might be able to drop a simple inline image tag and I can add some styling to make it responsive

i put the image in the 'assets' folder but would not resize, give i a shot

**Comment by stevep2007** [comment]
The marketing plan is to post regularly to the blog about our sponsors, sharing them on social media to increase attention and increase the number of applications to the hackathon.

We have a queue of blog posts for this that we want to post on Weds and Thursday to be reshared on SM

Posting to the website blog requires some preparation. 

1.) the main section of the blog needs a better image than the stock Zepplin image.
2.) the individual posts should resolve when shared on social media to an appropriately sized image by the editor and the H1 and if available H2.
3.) Editors should send posts with web banner images and FB and Twitter images appropriately sized. 
3:2 for web
2:1 for twitter
1.91:1 for facebook

3.) After the first post (pray tomorrow) the process is a handoff to Seha. 

4.) Max to identify any unique versioning to inline and links not used elsewhere in the zeplin framework.

**Comment by stevep2007** [comment]
@mcclaskiem Images cropped for the blog, twitter and facebook - medium identified in the name

![returning blog post](https://user-images.githubusercontent.com/3673316/50222747-2e036500-0367-11e9-9817-a255480cadec.jpg)
![returning twitter image](https://user-images.githubusercontent.com/3673316/50222748-2e036500-0367-11e9-86fc-b8388f0d99fd.jpg)
![returning facebook image](https://user-images.githubusercontent.com/3673316/50222749-2e036500-0367-11e9-8708-6550a2506541.jpg)

**Comment by stevep2007** [comment]
Here are the images shared in original format https://drive.google.com/open?id=1E3DqOF5R80t_05BdrwXrlyYdL7gfqqJ7
